src/Next.php

```diff
@@ -55,26 +55,26 @@ public static function run(string $versionString, bool $strict): string
         ///////////////////////////////////////////////////////////////////////////////////////////////
         // Raw versions
         ///////////////////////////////////////////////////////////////////////////////////////////////
-        $output  = '::set-output name=major::' . $version->incrementMajor() . PHP_EOL;
-        $output .= '::set-output name=minor::' . $version->incrementMinor() . PHP_EOL;
+        $output  = '"major=' . $version->incrementMajor() . '" >> $GITHUB_OUTPUT' . PHP_EOL;
+        $output .= '"minor=' . $version->incrementMinor() . '" >> $GITHUB_OUTPUT' . PHP_EOL;
 
         ///////////////////////////////////////////////////////////////////////////////////////////////
         // v prefixed versions
         ///////////////////////////////////////////////////////////////////////////////////////////////
-        $output .= '::set-output name=v_major::v' . $version->incrementMajor() . PHP_EOL;
-        $output .= '::set-output name=v_minor::v' . $version->incrementMinor() . PHP_EOL;
+        $output .= '"v_major=v' . $version->incrementMajor() . '" >> $GITHUB_OUTPUT' . PHP_EOL;
+        $output .= '"v_minor=v' . $version->incrementMinor() . '" >> $GITHUB_OUTPUT' . PHP_EOL;
```

Comment:
Change all of them in this file to this format, and then in `next.php` in the root of this repository replace the `echo` with a file_put_contents as in: https://github.com/WyriHaximus/github-action-supported-php-versions/blob/940773a095e0d019bca8d75262755dcfbb3092f1/versions.php#L27

```suggestion
        $output .= '"v_minor=v' . $version->incrementMinor() . "\"\n";
```